plugin/trino-kudu/src/test/java/io/trino/plugin/kudu/TestKuduConnectorTest.java

```diff
@@ -745,6 +745,26 @@ public void testUpdateAllValues()
         });
     }
 
+    /**
+     * This test fails intermittently because Kudu doesn't have strong enough
+     * semantics to support writing from multiple threads.
+     */
```

Comment:
I'd probably do a `throw new SkipException("<your_comment>")` here instead of disabling test.

Reply:
To be honest I just followed the style we had for other update test in this connector